apps/build_plant_metabolic_model/display.yaml

```diff
@@ -3,9 +3,9 @@
 #
 name     : Build Plant Metabolic Model
 subtitle : |
-    Build a metabolic model for a plant based on annotated sequences of transcripts.
+    Build a metabolic model for a plant based on annotated sequences of coding sequences.
```

Comment:
"annotated sequences of coding sequences" seems redundant. Maybe just "annotated coding sequences"?